src/docview.ts

```diff
@@ -69,7 +73,27 @@ export class DocViewProvider implements Disposable {
                 link.attribs.href = mkCommandUri('lean.openDocView', href);
             }
         }
-        this.getWebview().webview.html = $.html();
+        $(`<nav style="box-sizing: border-box; position : sticky; top : 0px; padding : 4px; background : white">
+        <a class="btn" href="${mkCommandUri('lean.backDocView')}">back</a>
+        <a class="btn" href="${mkCommandUri('lean.forwardDocView')}">forward</a>
```

Comment:
The `<nav>` bar has the advantage that it is always visible, which is nice.

 * Stylistically, how about `padding: 1ex; text-align: right;` for `<nav>`?
 * The buttons are not visually distinct from the content.  Maybe `border-bottom: 1px solid gray` or something?
 * Can we make the `<nav>` 100% in width?
 * Can we disable the back/forward links if the history is empty?
 * How about arrows in the links, i.e. `← back`, `forward →`?
 * The `<nav>` is shadowed by the `<pre>` blocks.

Reply:
LGTM.  I'm not sure if the title bar buttons are worth the trouble since they are only visible when you focus the documentation view.

If you have time, could you look into adding a URL bar?  This was the other feature that [Patrick really missed](https://leanprover.zulipchat.com/#narrow/stream/179818-Lean-Together.202019/topic/Guinea.20pigs.20wanted!/near/154947296).